checkstyle/suppressions.xml

```diff
@@ -17,8 +17,14 @@
               files="(ApiMessageType).java|MessageDataGenerator.java"/>
     <suppress checks="MethodLength"
               files="MessageDataGenerator.java"/>
+    <suppress checks="Regexp"
+              files="MessageGenerator.java"
+              lines="285,288"/>
 
     <!-- Clients -->
+    <suppress checks="Regexp"
+              files="Exit.java"
+              lines="43"/>
```

Comment:
It seems really brittle to have line numbers in the suppression. Could we:
1. Remove the `lines` part.
2. Give a name to the `Regexp` so that we only exclude the one related to `System.exit` (not sure if this is possible)

Reply:
I agree, @ijuma , but this is the best I could come up with. Apparently, it's not possible to give it a name, so the only alternative would be to suppress _all_ Regexp checks for this file.